libc/functions/m/meminfo.impl.md

```diff
@@ -0,0 +1,76 @@
+# Synopsis 
+`#include <sys/mman.h>`</br>
+
+` void meminfo(meminfo_t *info);`</br>
+
+## Description
+
+The `meminfo()` function shall assign values for a `meminfo_t` structure. The `meminfo_t` implementation is presented below.
+
+```C
+typedef struct _meminfo_t {
+    struct {
+        unsigned int alloc, free, boot, sz;
+        int mapsz;
+        pageinfo_t *map;
+    } page;
+
+    struct {
+        unsigned int pid, total, free, sz;
+        int mapsz, kmapsz;
+        entryinfo_t *kmap, *map;
+    } entry;
+} meminfo_t;
+```
+
+The structure consists of two following parts: page and entry.
+
+The page, in general, is a representation of RAM physical memory's part. For more information about paging technique please refer to the [`Memory management chapter`](../../../kernel/vm/README.md) and [`Page allocator chapter`](../../../kernel/vm/page.md).
+
+Each of the structure's elements has been briefly described:
+
+* `alloc` - Size of allocated pages in bytes,
+* `free` - Size of free pages in bytes,
+* `boot` - Size of pages which belong to a boot loader in bytes,
```

Comment:
I'd change the `boot` member description. It stores size of pages marked with `PAGE_OWNER_BOOT` flag. These pages don't belong to the boot loader. They are reserved by the internal boot firmware and store its data (e.g. BDA (BIOS Data Area) and EBDA (Extended BIOS Data Area) reserved by BIOS on ia32 or DTB (Device Tree Blob) reserved by the firmware operating in machine mode like BBL or OpenSBI on riscv64).

Reply:
Thanks, I was not sure about it, corrected.